src/test/java/com/puppycrawl/tools/checkstyle/meta/MetadataGeneratorUtilTest.java

```diff
@@ -82,7 +82,7 @@ public void testMetadataFilesGenerationAllFiles(@SystemOutGuard.SysOut Capturabl
             "45: " + getCheckMessage(MSG_DESC_MISSING, "AbstractClassCouplingCheck"),
             "26: " + getCheckMessage(MSG_DESC_MISSING, "AbstractAccessControlNameCheck"),
             "30: " + getCheckMessage(MSG_DESC_MISSING, "AbstractNameCheck"),
-            "29: " + getCheckMessage(MSG_DESC_MISSING, "AbstractParenPadCheck"),
+            "30: " + getCheckMessage(MSG_DESC_MISSING, "AbstractParenPadCheck"),
```

Comment:
For future reviewers: this needed to be updated because of adding a new import to `AbstractParenPadCheck`, line with violation moved + 1.